packages/legend-extension-external-language-morphir/src/ELMorphir_GenerationExtension.ts

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
You should have renamed this file and class to `ELMorphir_Extension`

Reply:
Fixed.